schema.graphql

```diff
@@ -564,3 +564,12 @@ type Liquidation @entity {
   fCashMaturities: [BigInt!]
   fCashNotionalTransfer: [BigInt!]
 }
+
+# Data accumulated and condensed into day stats for lending and borrowing
+type LendBorrowDayData @entity {
+  id: ID!
+  date: Int!
+  volumeNetUnderlyingCash: BigInt!
+  tradeType: TradeType!
```

Comment:
Depends on how you plan on querying this, but you can also consider flattening this object out and just include the lend borrow figures as separate fields. That might improve query performance for larger datasets because you end up scanning fewer rows. So instead of `tradeType` you have:
```
totalLendVolumeUnderlying
totalLendVolumeAsset
totalLendVolumefCash
totalLendVolumeTxnCount
totalBorrowVolumeUnderlying
totalBorrowVolumeAsset
totalBorrowVolumefCash
totalBorrowVolumeTxnCount
```

Reply:
Our goal is to have a stacked bar chart displaying the volume of borrow and lending over time. I don't think don't hurt performance keeping tradeType, do you still think I should flatten out the object ?